src/models/nftRewardTier.ts

```diff
@@ -3,6 +3,9 @@ import { CurrencyOption } from './currencyOption'
 
 export const NFT_METADATA_CONTRIBUTION_FLOOR_ATTRIBUTES_INDEX = 0
 
+export type DV1 = '1'
```

Comment:
**suggestion** i like the thinking here but i think it might be clearer to use a full name here JB721DelegateVersion_1 and JB721DelegateVersion_1_1 or similar